apps/projects/app/App.js

```diff
@@ -294,7 +294,13 @@ class App extends React.PureComponent {
       tokenArray,
       ipfsString,
       description
-    )
+    ).subscribe(
+      txHash => {
+        post({ variables: { body: `This issue has a bounty attached to it.\nAmount: ${issuesArray[0].size} autark\nDeadline: ${issuesArray[0].deadline}`,
```

Comment:
The bounty isn't necesarily always autark tokens, and I'd make sure that both the deadline and size are displayed in human readable formats. The decimals for the size will need to be inferred from the token.

Reply:
@Quazia, I made the changes you requested:
* **Token Symbol**: it now changes with every different token, as you requested.
* **Deadline format**: it's a JS date object. In this case, I believe the `.toUTCString()` method to be the most human-friendly, because the Github message won't change depending on the user's timezone.
* **Amount format**: this raises some sort of UX dilemma. If the token has 18 decimals, as many ERC-20 tokens do, *and* the amount isn't round, what is more human-friendly? to show the full 18 decimals for the sake of precision (we're talking money after all), or to round the amount to some fixed number of decimals? I don't pretend to solve that here; instead I'll just follow the [common practice](https://github.com/AutarkLabs/planning-suite/blob/dev/apps/projects/app/components/Panel/FundIssues/FundIssues.js#L212).

Reply:
Should actually post on each issue:

```
        issuesArray.forEach(issue => {
          post({ variables: {
            body: `This issue has a bounty attached to it.\nAmount: ${issue.size.toFixed(2)} ${bountySymbol}\nDeadline: ${issue.deadline.toUTCString()}`,
            subjectId: issue.key } })
        }),
        err => console.log(`error: ${err}`)
```